functions/yum/repo/gpgkeys/os_updates.pp

```diff
@@ -0,0 +1,9 @@
+# Build a list of GPG keys needed by a os_updates repo
+# @return Array[String]
+function simp::yum::repo::gpgkeys::os_updates() {
+  $release_key = $facts['os']['name'] ? {
+    'RedHat' => 'RPM-GPG-KEY-redhat-release',
+    default  => "RPM-GPG-KEY-${facts['os']['name']}-${facts['os']['release']['major']}"
+  }
+  [$release_key]
+}
```

Comment:
@lamawithonel Acknowledged, but it works and this will not be a breaking change to fix later. Please do not block on this.